Makefile

```diff
@@ -112,12 +112,12 @@ install: compile
 	$(MAKE) install_man
 
 install_man:
-	$(Q) mkdir -p $(PREFIX)/share/man/man1
+	$(Q) mkdir -p $(DESTDIR)$(PREFIX)/share/man/man1
```

Comment:
if $(DESTDIR) is  empty it  means we are installing to the current system instead to a package build tree; it's not a problem

*but* I just noted that for consistency with the rest of the Makefile I need to drop the extra slash from the lines below, wil amend the commit shortly